integration/common.go

```diff
@@ -47,8 +47,8 @@ var (
 
 func initImages(imageListFile string) {
 	imageList = ImageList{
-		Alpine:           "docker.io/library/alpine:latest",
-		BusyBox:          "docker.io/library/busybox:latest",
+		Alpine:           "ghcr.io/containerd/alpine:3.14.0",
+		BusyBox:          "ghcr.io/containerd/busybox:1.32",
```

Comment:
I see this file also references these: https://github.com/containerd/containerd/blob/5847340a7d2b30de5edb8c12a66cfe947d1b03fd/integration/image_list.sample.toml#L1-L2


But looking at https://github.com/containerd/containerd/commit/5847340a7d2b30de5edb8c12a66cfe947d1b03fd, I'm not sure it's actually used